docs/Installation.md

```diff
@@ -61,3 +61,14 @@ Note that in order for this to work for pip versions >= 21.3, your Git version m
 ## As a Docker image
 
 Use one of the Dockerfiles in the [docker](../docker) directory to build an image with an environment already set up and ready for running NNCF [sample scripts](../README.md#model-compression-samples).
+
+## Corresponding versions
+
+The following table lists the recommended corresponding versions of backend packages
+as well as the supported versions of Python:
+
+| NNCF    | OpenVINO   | PyTorch             | ONNX     | TensorFlow              | Python  |
+| ------- | ---------- | ------------------- | -------- | ----------------------- | ------- |
+| `2.6.0` | `2023.1.0` | `>=1.13.0`, `<2.1`  | `1.13.1` | `>=2.4.0`, `<=2.12.0`   | `>=3.7` |
```

Comment:
I don't think "it's recommended". We should recommend what is tested and according to https://github.com/openvinotoolkit/nncf#system-requirements: `This repository is tested on Python* 3.8.10, PyTorch* 2.0.1 (NVidia CUDA* Toolkit 11.7) and TensorFlow* 2.12.1 (NVidia CUDA* Toolkit 11.8).`

Reply:
As you know, Torch and TensorFlow has backward compatibility tests for provided versions, OpenVINO and ONNX does not have. Still, what should be considered the recommended versions, those that provide full validation?

Reply:
Updated PT and TF versions according [#system-requirements](https://github.com/openvinotoolkit/nncf#system-requirements):

Update current test requirements for PT: 
- `1.13.1` instead of `3.8.10`
- cuda `11.8` instead of `11.7`

Python: only 3.8